pkg/apis/cstor/v1/cstorpoolinstance.go

```diff
@@ -68,17 +68,17 @@ const (
 	// CStorPoolStatusEmpty ensures the create operation is to be done, if import fails.
 	CStorPoolStatusEmpty CStorPoolInstancePhase = ""
 	// CStorPoolStatusOnline signifies that the pool is online.
-	CStorPoolStatusOnline CStorPoolInstancePhase = "Healthy"
+	CStorPoolStatusOnline CStorPoolInstancePhase = "ONLINE"
```

Comment:
we have to update the `internalapis/apis` as well

Reply:
Well -- these constants are not present/required in the internal apis.

Reply:
For consistency, will push that -- and later we can decide whether we need those in internal apis or not.